speakerbox/src/test/java/com/mapzen/speakerbox/SpeakerboxTest.java

```diff
@@ -124,6 +124,14 @@ public void shouldReturnMuteState() throws Exception {
     }
 
     @Test
+    public void shouldStopOnMute() {
```

Comment:
On second thought... maybe its fine to always call `textToSpeech.stop()` from `mute()` since there is no harm in calling stop even when nothing is playing?